electron-app/src/utils.ts

```diff
@@ -89,7 +92,24 @@ export const createDir = (dirPath: string) => {
 
 // Get file data
 export const getFileData = (filePath: string, format: string = 'utf-8') => {
-  return fs.readFileSync(filePath, format);
+  const fileData = fs.readFileSync(filePath, format);
+  return formatConfigFileRead(fileData);
+};
+
+// Add squarebrackets masternode_operator in config file
+export const formatConfigFileRead = (fileData: string) => {
+  return fileData.replace(
+    new RegExp('masternode_operator', 'gi'),
+    'masternode_operator[]'
+  );
+};
+
+// Remove squarebrackets masternode_operator in config file
+export const formatConfigFileWrite = (fileData: string) => {
```

Comment:
Can you explain a bit what does this method do?

Reply:
Please refer to this message for the whole overview.
https://cakedefi.slack.com/archives/G017BGBTGER/p1614257741276500

Reply:
Hi @saurabh391, I believe that this works but it might be prone to issues. If we rely on string manipulation, especially on config files, we might remove configs accidentally or even existing configs that we have. What I suggest is to modify them on the electron side. I have encountered this problem as well and it's better to handle in the electron side.

We can pass the config in electron, decode it there. Once decoded, it will be a normal JS object and it's much safer to parse/modify the data instead of relying on string manipulation. Take a look at this example: https://github.com/DeFiCh/app/blob/55d753c945ceb633cd774059c1f69d97031bbef7/electron-app/src/controllers/wallets.ts#L56

What do you think @monstrobishi?

Reply:
Let's go with @saurabh391 solution for now and we can refactor in another update. We discussed this already, it needs some kind of workaround